src/test/java/HomeWork/DropdownList.java

```diff
@@ -14,9 +15,10 @@ public void dropdownList() {
         Assert.assertEquals(Rows.size(), 3);
         Rows.get(1).click();
         List<WebElement> Rows1 = driver.findElements(By.cssSelector("#dropdown > option"));
-        Assert.assertTrue(Rows1.get(1).getAttribute("selected").equals("true"));
+        Assert.assertTrue(Rows1.get(1).isSelected());
```

Comment:
:(
That's still absolutelly incorrect.
You should use Select class methods to work with dropdown like:
select.selectByVisibleText("Option 1");
Assert.assertEquals(select.getFirstSelectedOption().getText(), "Option 1")